packages/create-pwa/lib/index.js

```diff
@@ -96,6 +96,12 @@ module.exports = async () => {
                 return author;
             }
         },
+        {
+            name: 'template',
+            message: ({ name }) =>
+                `Which template would you like to use to bootstrap ${name}? Defaults to venia-concept.`,
+            default: 'venia-concept'
```

Comment:
Is there a difference between `@magento/venia-concept` and `venia-concept`? If not maybe we can use the namespaced version since it is similar to how a user would define by version

Reply:
We are maintaining an [array](https://github.com/magento/pwa-studio/blob/develop/packages/pwa-buildpack/lib/cli/create-project.js#L16) which will be [matched against first](https://github.com/magento/pwa-studio/blob/develop/packages/pwa-buildpack/lib/cli/create-project.js#L77). If that doesn't work, that's when the code looks to pull the package from NPM. The object that we have in that array is looking for `venia-concept`. You are right, what you mentioned is what the code does but right now the way the code is laid out works this way:

```
if template === 'venia-concept`
    // use local mapping to find the local venia-concept package
else if template === anything other than `venia-concept`
   // check inside mac cache and if it does not find that package, fetch from NPM
   // if template is `@magento/venia-concept` use the latest stable version
   // if template has a version string attached, use the package with that particular version.
```

If we change the default to `@magento/venia-concept` instead, we won't be able to use local packages while developing. There definitely will be a way to do that, but do we need to spend more time on that? I did think about what you mentioned but opted out personally, but wanted to check with the team. Thanks for bringing this up buddy. Good question.

Reply:
We discussed in dev time and I think we want to be pretty explicit here to avoid the "magic" that helped obscure the release bug. 

Basically the target template input text should be the deployed name of the package, as if you were doing a `yarn add` ie https://classic.yarnpkg.com/en/docs/cli/add/. So in this case, defaulting to `@magento/venia-concept` is fine, but it should default to fetch the latest release version from remote, _not_ local. To use local we would be explicit such as `template: file:/path/to/local/folder`.

I think the team had input so if you want to start a thread on slack about it, let's do it!

Reply:
Done, handled.